Automattic/jetpack
# Jetpack: Update QR post-publish panel to show Jetpack Icon

Part of https://github.com/Automattic/jetpack/issues/26048

Introduces Jetpack branding on post-publish panel


#### Changes proposed in this Pull Request:

* Adds Jetpack logo to post-publish QR Panel

![IMG_5922](https://user-images.githubusercontent.com/746152/189532627-9220d79f-f0eb-4154-bd09-fca92698a4af.jpg)



#### Other information:


#### Jetpack product discussion
p1HpG7-hDK-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a connected site
* Start writing a post
* Hit "Publish" , confirm with another click on Publish. and stop . Confirm in the right sidebar that you see the Jetpack logo 
  * QR Post
